### PR TITLE
Fix dead link in hello-world-in-your-browser.md

### PR DESCRIPTION
**Pull request by MacroChip**

### CODE REVIEW

File: content/guides/getstarted/hello-world-in-your-browser.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ re-execute your program.
 See the links below to learn more about writing Solana programs, and setting up
 your local development environment:
 
-- [Setup your local development environment](./setup-local-development.md)
+- [Setup your local development environment](https://solana.com/developers/guides/getstarted/setup-local-development)
```
Comment (nickfrosty) [inline content/guides/getstarted/hello-world-in-your-browser.md:317]: Great catch on this!

```suggestion
- [Setup your local development environment](/content/guides/getstarted/setup-local-development.md)
```

**Review by nickfrosty — CHANGES_REQUESTED** [review]
Great catch on this! But a better way is the absolute path within this repo. It will allow people to navigate to the file while exploring this repo, and on the respective page on solana.com/developers/guides/*

While you already have this PR open, can you fix the other `./*` links within this file?

**Comment by MacroChip** [comment]
I think that's it. Fyi I couldn't get next running properly locally to fully verify but I believe it's correct. The links were working properly in vscode.

**Comment by nickfrosty** [comment]
> I think that's it. Fyi I couldn't get next to compile the app properly to fully verify but I believe it's correct. The links were working properly in vscode.

what was the error you were getting when you tried to build it?
or are you referring to the GitHub action for `vercel` not completing?

**Comment by nickfrosty** [comment]
Thanks for the updates here! There was recently a huge docs migration merge into `main`, so I fixed the links for those changes

**Comment by MacroChip** [comment]
Very sorry I missed this notification. Build is working fine for me now! I don't remember what was happening.

**Pull request merged**